### DISCUSSION TITLE
Add `x_google_ignorelist`

### DISCUSSION THREAD

**Discussion by kirkstrobeck** [vercel/next.js#41371] [Ideas]
### Describe the feature you'd like to request

Per [Chrome 106 - What’s New in DevTools](https://www.youtube.com/watch?v=5gBqTXctxO8), please add `x_google_ignorelist`

### Describe the solution you'd like

See demo in video

### Describe alternatives you've considered

N/A

---

Thank you!

**Comment by 0xdevalias** [comment]
This would be useful. Here are some related issues on React/etc:

- https://github.com/facebook/react/issues/27774
- https://github.com/markerikson/react-prod-sourcemaps/issues/19
- https://github.com/vercel/next.js/issues/41370
- https://github.com/preactjs/preact/issues/4225
- https://github.com/sveltejs/svelte/issues/9740
- https://github.com/developit/microbundle/issues/1066
- https://github.com/angular/angular-cli/pull/23545
- https://nuxt.com/blog/v3-3#better-logging-in-browser-devtools
  - https://github.com/nuxt/nuxt/pull/19243
- https://vitejs.dev/config/server-options.html#server-sourcemapignorelist
  - https://github.com/vitejs/vite/pull/12174
  - https://github.com/vitejs/vite/pull/12633
  - https://github.com/vitejs/vite/pull/12694
- https://rollupjs.org/configuration-options/#output-sourcemapignorelist
  - https://github.com/rollup/rollup/issues/4847
  - https://github.com/rollup/rollup/pull/4846
  - https://github.com/rollup/rollup/pull/4848
  - https://github.com/rollup/rollup/pull/4926
- https://github.com/Rich-Harris/magic-string/issues/241
  - https://github.com/Rich-Harris/magic-string/pull/240
  - https://github.com/Rich-Harris/magic-string/pull/243

**Reply by 0xdevalias** [reply]
> - https://developer.chrome.com/blog/new-in-devtools-120/#ignore-list-spec
>   - > The [source maps specification](https://sourcemaps.info/spec.html) has [adopted](https://github.com/tc39/source-map-spec/pull/19) the `ignoreList` field instead of `x_google_ignoreList` and DevTools now supports the new name with a fallback for the old one. Frameworks and bundlers can now use the new field name.
> 
> _Originally posted by @0xdevalias in https://github.com/facebook/react/issues/27774#issuecomment-1836969520_
